src/components/stats/SoloCommits.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+
+import { Query } from 'react-apollo';
+import { gql } from 'apollo-boost';
+import client from '../../utils/GitHubGQL';
+
+class SoloCommits extends Component {
+  constructor(props) {
+    super();
+    this.state = {
+      startDate: new Date(props.startDate).toISOString(),
+      endDate: `${props.endDate}T23:59:59.999Z`,
+      username: props.username,
+      repository: props.repository,
+      winner: '',
+      highScore: 0,
+    };
+  }
+
+  componentDidMount() {
+    const { username } = this.state;
+    const { repository } = this.state;
+    const { startDate } = this.state;
+    const { endDate } = this.state;
+
+    client.query({
+      query: gql`
+      {
+        repository(owner: "${username}", name: "${repository}") {
```

Comment:
Owner should already be a string so no need to string interpolate it (remove brackets and dollar sign around these variables)

Reply:
i think GQL needs this, couldnt resolve.